ci/cpu/build.sh

```diff
@@ -61,8 +69,8 @@ conda list --show-channel-urls
 # FIX Added to deal with Anancoda SSL verification issues during conda builds
 conda config --set ssl_verify False
 
-pip install git+https://github.com/dask/dask.git@main
-pip install git+https://github.com/dask/distributed.git@main
+pip install git+https://github.com/dask/dask.git@2022.9.2
+pip install git+https://github.com/dask/distributed.git@2022.9.2
```

Comment:
Maybe in a follow-up PR, but would it make more sense to use the same logic as we do in `ci/gpu/build.sh`, or is there a reason I'm not aware for keeping `pip install`?

Reply:
Ah ok, that would be even better. Thanks Prem!